source/en/skins-system.rst

```diff
@@ -0,0 +1,163 @@
+   without any changes. The answer will also include additional property with :samp:`name` equal to **ely**.
+
+   .. code-block:: javascript
+
+      {
+        "id": "ffc8fdc95824509e8a57c99b940fb996",
+        "name": "ErickSkrauch",
+        "properties": [
+          {
+            "name": "textures",
+            "signature": "QH+1rlQJYk8tW+8WlSJnzxZZUL5RIkeOO33dq84cgNoxwCkzL95Zy5pbPMFhoiMXXablqXeqyNRZDQa+OewgDBSZxm0BmkNmwdTLzCPHgnlNYhwbO4sirg3hKjCZ82ORZ2q7VP2NQIwNvc3befiCakhDlMWUuhjxe7p/HKNtmKA7a/JjzmzwW7BWMv8b88ZaQaMaAc7puFQcu2E54G2Zk2kyv3T1Bm7bV4m7ymbL8McOmQc6Ph7C95/EyqIK1a5gRBUHPEFIEj0I06YKTHsCRFU1U/hJpk98xXHzHuULJobpajqYXuVJ8QEVgF8k8dn9VkS8BMbXcjzfbb6JJ36v7YIV6Rlt75wwTk2wr3C3P0ij55y0iXth1HjwcEKsg54n83d9w8yQbkUCiTpMbOqxTEOOS7G2O0ZDBJDXAKQ4n5qCiCXKZ4febv4+dWVQtgfZHnpGJUD3KdduDKslMePnECOXMjGSAOQou//yze2EkL2rBpJtAAiOtvBlm/aWnDZpij5cQk+pWmeHWZIf0LSSlsYRUWRDk/VKBvUTEAO9fqOxWqmSgQRUY2Ea56u0ZsBb4vEa1UY6mlJj3+PNZaWu5aP2E9Unh0DIawV96eW8eFQgenlNXHMmXd4aOra4sz2eeOnY53JnJP+eVE4cB1hlq8RA2mnwTtcy3lahzZonOWc=",
+            "value": "eyJ0aW1lc3RhbXAiOjE0ODYzMzcyNTQ4NzIsInByb2ZpbGVJZCI6ImM0ZjFlNTZmNjFkMTQwYTc4YzMyOGQ5MTY2ZWVmOWU3IiwicHJvZmlsZU5hbWUiOiJXaHlZb3VSZWFkVGhpcyIsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS83Mzk1NmE4ZTY0ZWU2ZDhlYzY1NmFkYmI0NDA0ZjhlYmZmMzQxMWIwY2I5MGIzMWNiNDc2ZWNiOTk2ZDNiOCJ9fX0="
+          },
+          {
+            "name": "ely",
+            "value": "but why are you asking?"
+          }
+        ]
+      }
+
+   By default textures proxying isn't used for this query. To enable it, add an additional GET parameter
+   :samp:`?proxy=true`.
+
+   The server will return an empty response with :samp:`204` status in case when textures aren't found.
+
+------------------------------------------------------------------------------------------------------------------------
+
+You can also pass a number of additional GET parameters when making any of the above requests. They will be used
+to analyze the usage of the service by different versions of the game.
+
+:version: The version of the protocol that the skins are being requested. This is currently version :samp:`2`,
+          i.e. you need to specify :samp:`version=2`.
+
+:minecraft_version: The version of Minecraft that the request is made from.
+
+:authlib_version: The version of the Authlib used. This option is relevant for Minecraft versions 1.7.6+,
+                  where a separate library is used to load skins rather than an in-game code.
+
+An example of a textures query with passing of parameters described above:
+
+.. code-block:: text
+
+   http://skinsystem.ely.by/textures/erickskrauch?version=2&minecraft_version=1.14.0&authlib_version=1.5.25
+
+Additional URLs
++++++++++++++++
+
+You can also perform a skin and cape request by passing the nickname through the GET parameter. This feature is used
+to pass analytical parameters on the game versions up to 1.5.2, when the nickname is simply appended to the end of the
```

Comment:
```suggestion
to pass analytical parameters of game versions up to 1.5.2, where the nickname is simply appended to the end of the
```
Same is for the Russian version: "в которых" instead of "когда"